sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveShowCreateTableSuite.scala

```diff
@@ -61,6 +44,30 @@ class HiveShowCreateTableSuite extends ShowCreateTableSuite with TestHiveSinglet
     }
   }
 
+  test("simple external hive table") {
+    withSQLConf(
+        SQLConf.LEGACY_CREATE_HIVE_TABLE_BY_DEFAULT_ENABLED.key -> "true") {
+      withTempDir { dir =>
+        withTable("t1") {
+          sql(
+            s"""CREATE TABLE t1 (
+               |  c1 INT COMMENT 'bla',
+               |  c2 STRING
+               |)
+               |LOCATION '${dir.toURI}'
+               |TBLPROPERTIES (
+               |  'prop1' = 'value1',
+               |  'prop2' = 'value2'
+               |)
+             """.stripMargin
+          )
+
+          checkCreateTable("t1")
+        }
+      }
+    }
+  }
+
   test("partitioned hive table") {
```

Comment:
What about this test case?

Reply:
Due to the parser rule (the order between `COMMENT` and `PARTITIONED BY`), it creates Hive table.

Reply:
Got it. It's really tricky. However, it will be broken in the future if we change Parser. Can we guard all test case with the conf?

Reply:
ok.

Reply:
Thanks~